examples/inference/anisotropy/submit_all.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Submit all SMC-RandomWalk inference jobs
+
+for dir in */; do
+    dir=${dir%/}  # Remove trailing slash
+
+    # Submit if submit.sh exists
+    if [ -f "$dir/submit.sh" ]; then
+        echo "Submitting $dir"
+        cd "$dir"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)